### DISCUSSION TITLE
Sonoff ZBMINI configre edge mode through MQTT?

### DISCUSSION THREAD

**Discussion by JohnBull-1975** [Koenkk/zigbee2mqtt#17072] [Devices]
Hi,
I have searched but not found much regarding if the sonoff ZB Mini (the one with neutral) is possible to configure for impulse switch through ZigBee2MQTT.
According to some web sites, Sonoff introuduced three different switching modes a while ago and one of them is edge, that looks to be possible to use with impulse witch (switch with a spring).
Some old discussion states that it is only possible to configure through sonoff app. But that discussion was from around 2020.
Anyone using the ZB MINI with ZigBee2MQTT in Home Assistant and can confirm if it's possible to set switch mode in there?

**Comment by ravextech** [comment]
for my rocker switches I solved the problem like this https://community.home-assistant.io/t/zbmini-in-zha-enable-edge-mode/407170/9
